e2e/suites/viewer/viewer-actions.test.ts

```diff
@@ -244,8 +245,16 @@ describe('Viewer actions', () => {
       await viewer.waitForViewerToOpen();
 
       await toolbar.openMoreMenu();
-      expect(await toolbar.menu.cancelEditingAction.isPresent()).toBe(true, `'Cancel Editing' button should be shown`);
-      expect(await toolbar.menu.editOfflineAction.isPresent()).toBe(false, `'Edit Offline' shouldn't be shown`);
+      await BrowserVisibility.waitUntilElementIsVisible(
```

Comment:
this would make the test fail with a timeout instead of an expectation failure.
you could use isPresentAndDisplayed inside an expect, since that returns a boolean

Reply:
is present doesn't deal with async display instead like that it wait